src/test/resources/pages/commonAssertRegexTestPage.html

```diff
@@ -28,7 +28,7 @@
         <div id="frameTopWrapper">
             <div id="logo">
                 <a href="/index.jsp">
-                <img width="150px" height="47px" src="/images/teammateslogo.jpg" alt="TEAMMATES" />
+                s<img width="150px" height="47px" src="/images/teammateslogo.jpg" alt="TEAMMATES" />
```

Comment:
this line is not intended to be committed   **s<img width="150px" height="47px" src="/images/teammateslogo.jpg" alt="TEAMMATES" />**  **s** before image tag was by mistake ignore this whie merging

Reply:
@shaharyar-shamshi, "ignore this while merging" is not an acceptable practice here. In fact it is explicitly stated in one of the items in the [PR template](https://github.com/TEAMMATES/teammates/blob/master/.github/PULL_REQUEST_TEMPLATE.md) that "[You should have] Gone through all the changes in this PR and ensured that no unintended changes were made".

If this kind of change is accidentally/otherwise introduced here, you need to revert it.

Reply:
okk sir this will be not be done again

Reply:
@wkurniawan07  sir should I make another pull request of this file . Or  Anything you would Suggest And Sorry for the mistake

Reply:
Don't worry about all the mistakes, they can always be corrected. As @chowyb has said before, there is no need to open another PR; just push more changes here in this PR.

Reply:
Okk sir  thanks sir 😅😅 this is the learning process for me and thanks for your support